pkg/registry/batch/job/strategy_test.go

```diff
@@ -580,6 +551,30 @@ func TestJobStrategyValidateUpdate(t *testing.T) {
 				job.Annotations["foo"] = "bar"
 			},
 		},
+		"deleting user annotation": {
+			job: &batch.Job{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:            "myjob",
+					Namespace:       metav1.NamespaceDefault,
+					ResourceVersion: "0",
+					Annotations: map[string]string{
+						batch.JobTrackingFinalizer: "",
+						"foo":                      "bar",
+					},
+				},
+				Spec: batch.JobSpec{
+					Selector:       validSelector,
+					Template:       validPodTemplateSpec,
+					ManualSelector: pointer.BoolPtr(true),
+					Parallelism:    pointer.Int32Ptr(1),
+				},
+			},
+			update: func(job *batch.Job) {
+				// change something.
```

Comment:
drop the comment, or make specific like `delete the user annotation`, but I don't see point in the comment since the testcase description says what it does

Reply:
Good callout

Reply:
done.